lib/services/livesync/ios-livesync-service.ts

```diff
@@ -33,10 +42,13 @@ class IOSLiveSyncService extends PlatformLiveSyncServiceBase<Mobile.IiOSDevice>
 	}
 
 	protected restartApplication(deviceAppData: Mobile.IDeviceAppData): IFuture<void> {
+		let projectData: IProjectData = this.$injector.resolve("projectData");
 		if (this.$options.debug) {
+			this.device.applicationManager.restartApplication(deviceAppData.appIdentifier, projectData.projectName).wait();
+			let appId = this.device.isEmulator ? projectData.projectName : deviceAppData.appIdentifier;
+			this.device.applicationManager.stopApplication(appId).wait();
```

Comment:
The first restart was necessary because the stopApplication command doesn't work correctly. It is addressed now.